tensorflow/compiler/xla/service/gpu/alpa_nccl_wrapper.cc

```diff
@@ -263,7 +282,21 @@ StatusOr< std::shared_ptr< std::vector<ncclComm_t> > > NcclCreateCommunicators(i
     XLA_CUDA_RETURN_IF_ERROR(ncclCommInitRank(comms.data()+i, world_size, nccl_uid, devices_global_rank[i]));
   }
   XLA_CUDA_RETURN_IF_ERROR(ncclGroupEnd());
-  return std::make_shared< std::vector<ncclComm_t> >(std::move(comms));
+
+  std::vector<cudaStream_t> streams;
+  streams.resize(n_devices);
+  for (int i=0; i<n_devices; i++) {
+    cudaSetDevice(devices_ids[i]);
+    if (nccl_use_multistream) 
```

Comment:
code style:
always use `{}` to wrap if-else block even it only has one line

Reply:
OK, get it.